### PR TITLE
Fix redirects and add missing guides

### PR DESCRIPTION
**Pull request by nachoiacovino**

### CODE REVIEW

File: guides/create-a-pack-with-typescript-and-nextjs/index.mdx [hunk 1]
```diff
@@ -45,6 +50,8 @@ This is what your project should look like now:
 
 ![thirdweb Packs Project](./thirdweb-dashboard.png)
 
+
+:::warning
```
Comment (jnsdls) [inline guides/create-a-pack-with-typescript-and-nextjs/index.mdx:54]: why is this warning here?

File: guides/create-a-pack-with-typescript-and-nextjs/index.mdx [hunk 2]
```diff
@@ -208,6 +215,8 @@ Go back to the thirdweb dashboard and open your "thirdweb Packs Bundle" module.
 
 ![Mint](./mint-nfts.png)
 
+
+:::warning
```
Comment (jnsdls) [inline guides/create-a-pack-with-typescript-and-nextjs/index.mdx:219]: why is this warning here?

File: guides/create-a-pack-with-typescript-and-nextjs/index.mdx [hunk 3]
```diff
@@ -218,6 +227,8 @@ When you have your LINK, go back to the thirdweb dashboard and open the "thirdwe
 
 ![Funding LINK](./funding-link.png)
 
+
+:::warning
```
Comment (jnsdls) [inline guides/create-a-pack-with-typescript-and-nextjs/index.mdx:231]: why is this warning here?

File: guides/dynamic-allowlist-edition/index.mdx [hunk 4]
```diff
@@ -0,0 +1,819 @@
...
+
+I am going to add a simple button for adding users to allowlist. You can even create an early access form or something similar. So, create a button in `index.tsx`:
+
+```tsx
+<button>Add wallet to allowlist</button>
+```
+
+Now, let's create an api to add the wallet to the allowlist. So, create a new file `add-to-allowlist.ts` and add the following:
+
+```ts:add-to-allowlist.ts
```
Comment (jnsdls) [inline guides/dynamic-allowlist-edition/index.mdx:503]: these need to be updated

File: guides/token-button-react/index.mdx [hunk 5]
```diff
@@ -1,6 +1,7 @@
 ---
 authors: raza
 tags: [react, token, button, front-end]
+draft: true
```
Comment (jnsdls) [inline guides/token-button-react/index.mdx:4]: does this do anything in docusaurus?

**Review by jnsdls — CHANGES_REQUESTED** [review]
small fixes

File: guides/token-button-react/index.mdx (hunk 5 above)
Comment (nachoiacovino) [reply guides/token-button-react/index.mdx:4]: yes, hides it in prod, this guide left hanging when moved to v2, we should update it/kill it at some point (not priority right now)

**Pull request merged**